lottie/src/main/java/com/airbnb/lottie/model/content/LBlendMode.java

```diff
@@ -0,0 +1,92 @@
+package com.airbnb.lottie.model.content;
+
+import android.os.Build;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import androidx.core.graphics.BlendModeCompat;
+
+/**
+ * Lottie BlendMode,
+ * not to be confused with Paint.BlendMode in android graphics core,
+ * which we will rely on for rendering.
+ */
+public enum LBlendMode {
+  NORMAL,
+  MULTIPLY,
+  SCREEN,
+  OVERLAY,
+  DARKEN,
+  LIGHTEN,
+  COLOR_DODGE,
+  COLOR_BURN,
+  HARD_LIGHT,
+  SOFT_LIGHT,
+  DIFFERENCE,
+  EXCLUSION,
+  HUE,
+  SATURATION,
+  COLOR,
+  LUMINOSITY,
+  ADD,
+  HARD_MIX;
+
+  @Nullable
+  public BlendModeCompat toNativeBlendMode() {
+    switch (this) {
+      case NORMAL:
+        return null; // BlendMode.CLEAR?
```

Comment:
Can you elaborate what this comment is for? I think null seems right for NORMAL, not CLEAR.

Reply:
I wasn't quite sure. Have gone back and confirmed that `null` is the way to go